pytest.ini

```diff
@@ -0,0 +1,5 @@
+[pytest]
+python_files = test_*.py
+python_functions = test_*
+filterwarnings =
+    ignore::DeprecationWarning:traitlets.*:
```

Comment:
[Note for myself] re-check if the warning is suppressed locally, may need further changes (`pytest -W error -W 'ignore: :DeprecationWarning'` seems to work, the warning is raised from dependencies from k3d)